cli/script/command-executor.ts

```diff
@@ -1149,58 +1167,64 @@ export var release = (command: cli.IReleaseCommand): Promise<void> => {
 }
 
 export var releaseCordova = (command: cli.IReleaseCordovaCommand): Promise<void> => {
-    var platform: string = command.platform.toLowerCase();
-    var projectRoot: string = process.cwd();
-    var platformFolder: string = path.join(projectRoot, "platforms", platform);
-    var platformCordova: string = path.join(platformFolder, "cordova");
-    var outputFolder: string;
-
-    if (platform === "ios") {
-        outputFolder = path.join(platformFolder, "www");
-    } else if (platform === "android") {
-        outputFolder = path.join(platformFolder, "assets", "www");
-    } else {
-        throw new Error("Platform must be either \"ios\" or \"android\".");
-    }
+    var releaseCommand: cli.IReleaseCommand = <any>command;
+    return checkIfAppExists(command.appName)
```

Comment:
I think it's better to put such validation before preparing update contents because it could save time - user have not to wait until `cordova prepare` or  `react-native bundle`  tasks finished.